render/detailed/connections.go

```diff
@@ -0,0 +1,197 @@
+package detailed
```

Comment:
Also, it seems like the functions have a bit of redundancy in the names `buildThis` and `makeThat`